src/styles/main.scss

```diff
@@ -1,3 +1,26 @@
+@import "./utils/variables";
+@import "./utils/mixin";
+
+@import "./blocks/header";
+@import "./blocks/cards";
+@import "./blocks/stars.scss";
+
+* {
+  box-sizing: border-box;
+
+  font-family: $main-font-family;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+  padding: 0;
```

Comment:
It's the initial value for tag `p`
so, `p` doesn't have a padding by default value
![image](https://github.com/mate-academy/layout_catalog/assets/91411692/1341c74a-fa9f-4e0b-95fd-56d88ac109b8)